README.md

```diff
@@ -1,13 +1,17 @@
 # GitHub Tracker
 
-Replace this readme with your own information about your project.
+This week I create a place to keep track of the GitHub repos.
+HTML,CSS and javascript.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
 
 ## The problem
+I started with fetching the information  through API.
+then I tried to figure out how to filter the information I needed.
+When I had information i displayed it on the site.
+Styled with CSS (mobile first).
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+If I had more time I would try to include a search bar and also which languages the projects includes.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://silly-mestorf-a038ed.netlify.app/
```

Comment:
If you would like to change the name of the site you can do this on Netlify if you go to the site overview > site settings > and under site information > change site name